src/i_oalsound.h

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright(C) 2023 Roman Fomin
+// Copyright(C) 2023 ceski
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//      System interface for OpenAL sound.
+//
+
+#ifndef __I_OALSOUND__
+#define __I_OALSOUND__
+
+#include "alext.h"
+
+#include "doomtype.h"
+#include "i_sound.h"
+
+#define OAL_CULL_DISTANCE 1200 // Distance where sound is culled.
```

Comment:
Could we factor these out into `i_sound.h` so that both `i_oalsound.c` and `i_mbfsound.c` use the same values (modulo frac conversion, that is)?

Reply:
Something like this? https://github.com/fabiangreffrath/woof/pull/1118/commits/8facdde742a97242796751495be732f9086aed49

Reply:
Yes, looks good. I guess the precompiler will take care of the bit-shifting left and right within the macros.